mentions/src/main/kotlin/io/spine/examples/pingh/mentions/RemoteGitHubSearch.kt

```diff
@@ -81,36 +81,41 @@ public class RemoteGitHubSearch(engine: HttpClientEngine) : GitHubSearch {
      * @param token The `PersonalAccessToken` to access user's private repositories.
      * @param updatedAfter The time after which GitHub items containing the searched mentions
      *   should have been updated.
+     * @param onlyOnFirstPage If `true`, retrieves mentions only from the first page of results.
+     *   If `false`, fetches all mentions since [updatedAfter].
      * @see [GitHubSearch.searchMentions]
      */
     @Throws(CannotObtainMentionsException::class)
     public override fun searchMentions(
         username: Username,
         token: PersonalAccessToken,
-        updatedAfter: Timestamp
+        updatedAfter: Timestamp,
+        onlyOnFirstPage: Boolean
```

Comment:
Instead, you should have an optional parameter `limit` (an integer number), which would mean "return no more than $limit of most recent mentions" if set.

The name is chosen to sound familiar to those who previously have used SQL.